MovieApp/MovieApp/App/Features/Movies/Components/SearchBar/SearchBarStackView.swift

```diff
@@ -0,0 +1,57 @@
+import UIKit
+
+class SearchBarStackView: UIStackView {
+    
+    let defaultInset = 11
+    let secondaryInset = 15
+    let magnifierSize = CGSize(width: 21, height: 21)
+    let cancelSize = CGSize(width: 13, height: 13)
+    
+    var searchView: UIView!
+    var magnifierImageView: UIImageView!
+    var searchTextField: UITextField!
+    var searchCancelButton: UIButton!
+    var cancelButton: UIButton!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        buildViews()
+        searchCancelButton.addTarget(self, action: #selector(didTapSearchCancelButton), for: .touchUpInside)
+        searchTextField.addTarget(self, action: #selector(didTapTextField), for: .touchDown)
+        cancelButton.addTarget(self, action: #selector(didTapCancelButton), for: .touchUpInside)
+    }
+    
+    required init(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    @objc func didTapSearchCancelButton() {
```

Comment:
check for all func that are not used inside the class where they are defined. Every func like that should be private